.tugboat/config.yml

```diff
@@ -0,0 +1,39 @@
+services:
+  # Configure a standard apache webserver to host our site.
+  apache:
```

Comment:
Do we generally use Apache? Not objecting, but would normally expect nginx

Reply:
I was referencing https://docs.tugboatqa.com/starter-configs/examples/static-html/ & https://docs.tugboatqa.com/starter-configs/examples/gatsby/ which are both using the `tugboatqa/httpd` image and `apache`. we can definitely change this if needed